test/Interop/C/bounds-safety/Inputs/bounds-attributed-struct.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include <ptrcheck.h>
+
+struct a {
+    int * __counted_by(len) a;
+    int len;
+};
+struct a * a(struct a);
```

Comment:
nit: 
```suggestion
struct a *a(struct a);
```

(Otherwise at first glance it looks a bit like `a * a`)